frontend/src/features/intent-draft-read/ui/IntentDetailPanel.module.css

```
@@ -0,0 +1,180 @@
+
+.cardBody {
+  padding: 10px 14px 16px;
+}
+
+.value {
+  font-size: 16px;
+  font-weight: 330;
+  line-height: 1.45;
+  letter-spacing: -0.14px;
+  color: var(--text-primary);
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)